app/src/main/java/org/sopt/sample/HomeAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package org.sopt.sample
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.sample.databinding.ItemHomeBinding
+import org.sopt.sample.databinding.ItemHeaderBinding
+
+
+
+    public class HomeAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+    private lateinit var itemHeaderBinding: ItemHeaderBinding
+    private lateinit var itemHomeBinding: ItemHomeBinding
+    val userList = mutableListOf<UserData>()  //userdata의 리스트를 userlist로 받음
+    val Header = 0 // 현재 뷰 객체가 타이틀임을 감지하기 위한 변수
+    val Body = 1
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        return if(viewType == Header) {
+            itemHeaderBinding = ItemHeaderBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+            HeaderViewHolder(itemHeaderBinding)
+        } else {
+            itemHomeBinding= ItemHomeBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+            HomeViewHolder(itemHomeBinding)
+        }
+    }
+
+    override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
+        if (holder is HomeViewHolder) {
+            holder.onBind(userList[position])
+        } else if (holder is HeaderViewHolder) {
+            holder
+        }
```

Comment:
```suggestion
        }
```

로직이 없다면 그냥 지워도 될 것 같아